### DISCUSSION TITLE
[Feature Request] Created/Starred filter view

### DISCUSSION THREAD

**Discussion by kkweon** [community/community#21664] [New to GitHub]
<p>There are currently two activities:</p>
<ul>
<li>A starred some/repo</li>
<li>A created a repository some/repo</li>
</ul>
<p>Sometimes, I want to view “Repo creation” activities only instead of viewing “All activity.”</p>
<p><img src="https://screenshot.googleplex.com/gKfvoTe5TAV.png" alt=""></p>
<p>Can we have add filter option, such that I can view “repo creation” activities or “star” activities only?</p>

**Comment by MChevy422** [comment, marked as answer]
<p>Hello <span><a class="mention" href="/kkweon">@kkweon</a></span> <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your input!<br><br>Greatly appreciated<br><br>MChevy422</p>
